packages/react-core-notifications/src/InAppMessaging/hooks/useMessage/__tests__/useMessage.spec.ts

```diff
@@ -177,7 +177,8 @@ describe('useMessage', () => {
 
     beforeEach(() => {
       jest.clearAllMocks();
-
+      Amplify.configure(mockawsmobile);
+      initializeInAppMessaging();
```

Comment:
Why do we need these?

Reply:
The calls to useMessage require initializing in-app :(

Reply:
Can we remove these calls by mocking?

Reply:
Yes missed initializeInApp, but to my understanding Amplify.configure is being mocked here 
https://jestjs.io/docs/mock-functions#mocking-modules